lms/static/js/spec/search/search_spec.js

```diff
@@ -0,0 +1,513 @@
+define([
+    'jquery',
+    'sinon',
+    'backbone',
+    'js/common_helpers/template_helpers',
+    'js/search/views/search_form',
+    'js/search/views/search_item_view',
+    'js/search/views/search_list_view',
+    'js/search/models/search_result',
+    'js/search/collections/search_collection',
+    'js/search/search_router',
+    'js/search/search_app'
+], function(
+    $,
+    Sinon,
+    Backbone,
+    TemplateHelpers,
+    SearchForm,
+    SearchItemView,
+    SearchListView,
+    SearchResult,
+    SearchCollection,
+    SearchRouter,
+    SearchApp
+) {
+    'use strict';
+
+    describe('SearchForm', function () {
```

Comment:
It would be preferable if all of these tests lived under one ``describe('Search')`` function. This would allow them all to be run as a group in the debugger.